de.tubs.variantsync.core/src/de/tubs/variantsync/core/managers/persistence/CodeMappingFormat.java

```diff
@@ -35,7 +36,7 @@ public CodeMappingFormat(IProject project) {
 	}
 
 	@Override
-	public IPersistentFormat<List<SourceFile>> getInstance() {
+	public APersistentFormat<List<SourceFile>> getInstance() {
```

Comment:
I do not understand this change.

Reply:
Because of changes in featureide,
IPersistentFormat.getInstance gets further specialized into APersistentFormat.getInstance
and cant be regeneralized later on.
AXMLFormat has unecessary double inheritance, it inherits from IPersistentFormat and extends APersistentFormat, so it is unclear what to implement.